runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterTrap.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2017, Robin Weymans <Robin.weymans@gmail.com>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.hunter;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Objects;
+import lombok.Getter;
+import lombok.Setter;
+import net.runelite.api.GameObject;
+
+/**
+ * Wrapper class for a GameObject that represents a hunter trap.
+ */
+class HunterTrap
+{
+	/** 
+	 * A hunter trap stays up 1 minute before collapsing. 
+	 */
+	private static final Duration TRAP_TIME = Duration.ofMinutes(1);
+	
+	/**
```

Comment:
I don't think that the getter and setter annotations are used inline like this anywhere else in RuneLite, so probably just add them on separate lines to follow conventions used elsewhere? (I am pro for inline annotations like this, as it is suggested to use by Google Java guidelines, but best is to follow project conventions).